Ledger_iOS/Models/PersistenceManager.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PersistenceManager.swift
+//  Ledger_iOS
+//
+//  Created by Elisha Chirchir on 13/11/2021.
+//
+
+import Foundation
+
+class PersistenceManager {
+    
+    private let defaults = UserDefaults.standard
+    
+    private let recordsKey = "records_key"
+    
+    var records: [Record]? {
+        get { return defaults.array(forKey: recordsKey) as? [Record]}
+        set { defaults.set(newValue, forKey: recordsKey)}
+    }
+}
```

Comment:
Why not just keep the old file and reuse it? No big deal

Reply:
I just figured all model-related code should be in the models package?